api/api.go

```diff
@@ -37,6 +39,43 @@ type SubmissionInfo struct {
 	State string `json:"state"`
 }
 
+// StatusInfo contains information about a user's status on a particular language track.
+type StatusInfo struct {
+	Language string `json:"track_id"`
```

Comment:
That seemed to be the variable used when describing the language when the API endpoint URLs were being build in many of the functions - specifically `Fetch` (line 83), `Submission` (line 155), `List` (line 240) & `Skip` (line 309). However, after doing a search in the project I now see that `TrackID` is used in the other custom types created for `Problem` and `Submission`. Maybe it would be beneficial to change all of those variables using `language` in the functions for API calls to `trackId` to be consistent?